tests/factories/lti_role.py

```diff
@@ -0,0 +1,28 @@
+import factory
+from factory import Faker
+from factory.alchemy import SQLAlchemyModelFactory
+
+from lms import models
+
+
+class LTIRole(SQLAlchemyModelFactory):
+    class Meta:
+        model = models.LTIRole
+
+        exclude = ("primary_role", "sub_role")
+
+    primary_role = Faker(
+        "random_element",
+        elements=[
+            "http://purl.imsglobal.org/vocab/lis/v2/membership/Administrator",
+            "http://purl.imsglobal.org/vocab/lis/v2/membership/ContentDeveloper",
+            "http://purl.imsglobal.org/vocab/lis/v2/membership/Learner",
+            "http://purl.imsglobal.org/vocab/lis/v2/membership/Mentor",
+            "http://purl.imsglobal.org/vocab/lis/v2/membership/Manager",
+            "http://purl.imsglobal.org/vocab/lis/v2/membership/Office",
+        ],
+    )
+
+    sub_role = Faker("hexify", text="SubRole-^^^^^^^")
+
+    value = factory.LazyAttribute(lambda o: f"{o.primary_role}#{o.sub_role}")
```

Comment:
We need to be careful we don't accidentally generate two role strings the same, or it will break the uniqueness requirement. This is _unlikely_ to happen here, but is still possible.

Should this be an incrementing value instead?